src/foundry/client/data/collections/folder.d.mts

```diff
@@ -8,20 +9,26 @@ declare global {
    * @see {@link Folder} The Folder document
    */
   class Folders extends WorldCollection<typeof foundry.documents.BaseFolder, "Folders"> {
-    constructor(data?: Document.Stored<Folder.ConfiguredInstance>["_source"][]);
+    static documentName: "Folder";
 
     /**
      * Track which Folders are currently expanded in the UI
      */
-    protected _expanded: Partial<Record<string, boolean>>;
-
-    static documentName: "Folder";
+    _expanded: Partial<Record<string, boolean>>;
 
-    render(force?: boolean, context?: ApplicationOptions): void;
+    _onModifyContents<A extends DatabaseAction>(
+      action: A,
+      documents: Folder.ConfiguredInstance[],
+      result: AnyObject[] | readonly string[],
```

Comment:
Probably unimportant but is `result` really going to accept any object in the array?

Reply:
`result` is the document update data from the server. It's probably technically BaseFolder.UpdateData? It gets forwarded to the relevant world collection app to decide if it needs to re-render, e.g. if a folder name changed